engine/src/flutter/ci/builders/linux_web_engine_test.json

```diff
@@ -194,13 +194,6 @@
             "check-licenses"
           ],
           "script": "flutter/lib/web_ui/dev/felt"
-        },
-        {
-          "name": "web engine analysis",
-          "parameters": [
-            "analyze"
-          ],
-          "script": "flutter/lib/web_ui/dev/felt"
```

Comment:
Do we also need to update the generator script?

Reply:
Oh yeah definitely